gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-all.zip
```

Comment:
Doesn't the binary and .bat file change as well from 7.1 to 7.6?

Reply:
I don't know when this changes. There is a command to change wrapper version and I don't do that manually